.gitmodules

```diff
@@ -0,0 +1,15 @@
+[submodule "modules/cpp-common"]
+	path = modules/cpp-common
+	url = git@github.com:metaswitch/cpp-common.git
+[submodule "modules/gmock"]
+	path = modules/gmock
+	url = git@github.com:Metaswitch/gmock-upstream.git
+[submodule "modules/libevent"]
+	path = modules/libevent
+	url = git@github.com:libevent/libevent.git
+[submodule "modules/libmemcached"]
+	path = modules/libmemcached
+	url = git@github.com:Metaswitch/libmemcached-upstream.git
+[submodule "modules/sas-client"]
+	path = modules/sas-client
+	url = git@github.com:metaswitch/sas-client.git
```

Comment:
Afraid not. Even if you use [mock_sas.h](https://github.com/Metaswitch/cpp-common/blob/master/test_utils/mock_sas.h) you still need a real `sas.h` so you have to use the submodule.